lightblue/src/test/java/org/esbtools/eventhandler/lightblue/testing/SlowDataLightblueClient.java

```diff
@@ -138,6 +138,9 @@ public LightblueBulkDataResponse bulkData(DataBulkRequest request)
 
             throw new RuntimeException(e);
         } catch (Exception e) {
+        	if (e instanceof LightblueException) {
```

Comment:
nitpick: mix of spaces and tabs. Might be a good idea to set your editor to use spaces only and no tab characters.